docs/manual/fread_examples.rst

```diff
@@ -0,0 +1,490 @@
+
+
+    fread("data.zip/mtcars.csv")
+
+Read from `.xlsx` files - `.xls` files are not supported::
+
+    fread("excel.xlsx")
+
+For excel files, you can specify the sheet to be read::
+
+    fread("excel.xlsx/Sheet1")
+
+Note: `xlrd <https://pypi.org/project/xlrd/>`_ must be installed to read in excel files.
+
+You can also read in data from the command line. Simply pass the command line statement to the `cmd` parameter::
+
+     #https://blog.jpalardy.com/posts/awk-tutorial-part-2/
+     #You specify the `cmd` parameter
+     #Here we filter data for the year 2015
+     fread(cmd = """cat netflix.tsv | awk 'NR==1; /^2015-/'""")
+
+The command line can be very handy with large data; you can do some of the preprocessing before reading in the data to `datatable`.
+
+Dealing with Unicode Data
+-------------------------
+
+`Fread` is pretty powerful and can automatically detect the encoding in a number of cases.::
+
+    data = ("num,name\n"
+            "1,alïce\n"
+            "2,bøb\n"
+            "3,cárol\n"
+            )
+
+    fread(data)
+
+              num       name
+        0	1	alïce
+        1	2	bøb
+        2	3	cárol
+
+You can also specify the encoding; `fread` will recode into `UTF-8` before reading.
+Check `here <https://docs.python.org/3/library/codecs.html#standard-encodings>`_ for Python's list of standard encodings.
+
+Detect Thousand Separator
+-------------------------
+
+Thousand separators are automatically detected::
+
+    data = """Name|Salary|Position
+             James|256,000|evangelist
+            Ragnar|1,000,000|conqueror
+              Loki|250360|trickster"""
+
+    fread(data)
+
+        Name	Salary	Position
+    0	James	256000	evangelist
+    1	Ragnar	1000000	conqueror
+    2	Loki	250360	trickster
+
+Specify the Delimiter
+---------------------
+
+You can specify the delimiter via the `sep` parameter.
+Note that the  separator must be a single character string; non-ASCII characters are not allowed as the separator, as well as any characters in ``["'`0-9a-zA-Z]``::
+
+    data = """
+           1:2:3:4
+           5:6:7:8
+           9:10:11:12
+           """
+
+    fread(data, sep=":")
+
+    	C0	C1	C2	C3
+    0	1	2	3	4
+    1	5	6	7	8
+    2	9	10	11	12
+
+Dealing with Null Values and Blank Rows
+---------------------------------------
+
+You can pass a list of values to be treated as null, via the `na_strings` parameter::
+
+    data = """
+           ID|Charges|Payment_Method
+           634-VHG|28|Cheque
+           365-DQC|33.5|Credit card
+           264-PPR|631|--
+           845-AJO|42.3|
+           789-KPO|56.9|Bank Transfer
+           """
+
+    fread(data, na_strings=['--', ''])
+
+        ID	    Charges  Payment_Method
+    0	634-VHG	    28	     Cheque
+    1	365-DQC	    33.5     Credit card
+    2	264-PPR	    631	     NA
+    3	845-AJO	    42.3     NA
+    4	789-KPO	    56.9     Bank Transfer
+
+
+For rows with null values, set `fill=True`::
```

Comment:
Better say "incomplete rows", i.e. rows with less values than in other rows. This is different from null values.